### PR TITLE
Sidebar: Show / Hide on pages as required

### PR DESCRIPTION
**Pull request by cleargif**
#### What
- Introducing a new partial to cut down on duplication
- Apply logic to show / hide the sidebar as required

#### Ticket
[Sidebar: Show / Hide on pages as required](https://www.pivotaltracker.com/story/show/156087810)

#### Why
The side bar should only be shown on some pages.

### CODE REVIEW

File: app/views/external_users/claims/interim_fee/_fields.html.haml [hunk 1]
```diff
@@ -3,6 +3,7 @@
   %h2.bold-medium
     = t('.interim_fees')
   %p.form-hint.xsmall
+    = "All fees should be entered exclusive of VAT. The system will add VAT to fees for VAT registered users."
```
Comment (lostie) [inline app/views/external_users/claims/interim_fee/_fields.html.haml]: ❓ isn't this the same as the line below (that is using the locales)? 😕  (💭  If it's not, I would still consider defining a locale for this, avoiding static wording in the views).

File: config/locales/en.yml [hunk 2]
```diff
@@ -751,7 +751,7 @@ en:
             fixed_fees: "Fixed fees"
             fee_type: "Fee type"
             total: "Total"
-            prompt_text_html: Check the <a href="https://www.gov.uk/government/publications/graduated-fee-calculators" target="_blank" rel="external" title="Graduated Fee Calculator">graduated fee calculators</a> for help
+            prompt_text_html: Check the <a href="https://www.gov.uk/government/publications/graduated-fee-calculators" target="_blank" rel="external" title="Graduated Fee Calculator">graduated fee calculators</a> for help.
```
Comment (lostie) [inline config/locales/en.yml]: **[not a blocker]** Seems like this wording is the same across all different types of claim (we're repeating the same wording for each claim type that requires it).

💭  Consider this instead:

```ruby
prompt_text_html: &graduated_fee_prompt_content Check the <a
href="https://www.gov.uk/government/publications/graduated-fee-calculators" target="_blank" rel="external" title="Graduated Fee Calculator">graduated fee calculators</a> for help.
...
graduated_prompt_text: *graduated_fee_prompt_content
...
interim_fees_prompt_text_html: *graduated_fee_prompt_content
...
transfer_fees_prompt_text_html: *graduated_fee_prompt_content
```

This should work, and avoid repeating the same content across multiple prompts.

**Review by lostie — CHANGES_REQUESTED** [review]
If we could tied up those small nits, would be nice to make things easier to change in the future.

Other than that, LGTM 👍

File: app/views/external_users/claims/interim_fee/_fields.html.haml (hunk 1 above)
Comment (cleargif) [reply app/views/external_users/claims/interim_fee/_fields.html.haml]: ooops.. forgot to wire it up again!

File: config/locales/en.yml (hunk 2 above)
Comment (cleargif) [reply config/locales/en.yml]: cool .. will do like above.

**Review by lostie — APPROVED** [review]
LGTM 👍

**Pull request merged**